state-chain/pallets/cf-elections/src/electoral_systems/mocks.rs

```diff
@@ -413,6 +413,13 @@ register_checks! {
 			"Expected the election id to be incremented.",
 		);
 	},
+	election_id_updated_by(pre_finalize, post_finalize, update: impl Fn(ElectionIdentifierOf<ES>) -> ElectionIdentifierOf<ES> + Clone + 'static) {
```

Comment:
could make this more specifically `election_id_extra_incremented_by_one`, saves duping the logic in the tests